pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,23 +1,50 @@
-{ stdenv, lib, fetchurl, python2Packages, gettext }:
-
+{ stdenv, lib, fetchFromGitHub, fetchpatch, python2, gettext }:
+let
+  # pin requests version until next release.
+  # see: https://github.com/linkcheck/linkchecker/issues/76
+  python2Packages = (python2.override {
+    packageOverrides = self: super: {   
+      requests = super.requests.overridePythonAttrs(oldAttrs: rec {
+        version = "2.14.2";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "0lyi82a0ijs1m7k9w1mqwbmq1qjsac35fazx7xqyh8ws76xanx52";
+        };
+      });
+    };
+  }).pkgs;
+in
 python2Packages.buildPythonApplication rec {
-  name = "LinkChecker-${version}";
-  version = "9.3";
+  pname = "LinkChecker-${version}";
+  version = "9.3.1";
+
+  propagatedBuildInputs = (with python2Packages; [ 
+    requests
+  ]) ++ [ gettext ];
 
-  buildInputs = with python2Packages ; [ pytest ];
-  propagatedBuildInputs = with python2Packages ; [ requests ] ++ [ gettext ];
+  checkInputs = with python2Packages; [ pytest ];
 
-  src = fetchurl {
-    url = "mirror://pypi/L/LinkChecker/${name}.tar.gz";
-    sha256 = "0v8pavf0bx33xnz1kwflv0r7lxxwj7vg3syxhy2wzza0wh6sc2pf";
+  # the original repository is abandoned, development is now happening here:
+  src = fetchFromGitHub {
+    owner = "linkcheck";
+    repo = "linkchecker";
+    rev = "v${version}";
```

Comment:
@dotlambda 
should "version" contain the "v" for the tag so it shows up in "${pname}-${version}" or is it allright this way?

Reply:
No that should stay as it is.
The two major  advantages of having `pname` and `version` are
- we can use `fetchPypi` which expects `pname` and `version` (without a "v" prefix most of the time)
- easier overriding because we do not need to override `name` nor `pname`